src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Device;
+using System.Device.I2c;
+using System.IO;
+using System.Numerics;
+using System.Text;
+using System.Threading;
+
+namespace Iot.Device.LidarLiteV3
```

Comment:
Please change namespace so that it matches for both HC-SR04 and this sensor (HC-SR04 we should change it too but not necessarily in this PR). The problem with having the same class name as namespace name is that it forces users to type both class name and namespace name since name becomes ambiguous which makes it kinda annoying to use

Reply:
I went with `TimeOfFlight` which seems like a common name among these devices, but open for suggestions. https://en.wikipedia.org/wiki/Time_of_flight

I also ended struggling a bit with namespacing issues with all the supporting classes.  Went with a `...TimeOfFlight.Models.LidarLiteV3` namespace to avoid conflict with the other devices, but not sure if I really like this.

Reply:
DistanceSensor perhaps, no strong opinion

Reply:
No strong opinion either.  That works, will change.